### DISCUSSION TITLE
https://fhir.hl7.at/r5-ELGA-e-medikation-main/intro_de_ballot.html

### DISCUSSION THREAD

**Discussion by giscus** [HL7Austria/ELGA-FHIR-e-Medikation-R5#4] [Comment]
# https://fhir.hl7.at/r5-ELGA-e-medikation-main/intro_de_ballot.html

https://fhir.hl7.at/r5-ELGA-e-medikation-main/intro_de_ballot.html

**Comment by Nik-xT** [comment]
### passage in question
Die Daten sind in einem oder mehreren computergestützten Informationssystemen gespeichert

### Describe your issue
Daten können auch in keinem System gespeichert sein!

### Describe your proposed solution
Der Leitfaden sollte nicht gespeichert werden.

### Rating
heavy disapproval

**Reply by Nik-xT** [reply]
Much true.

**Reply by Nik-xT** [reply]
Second much true.

**Reply by Nik-xT** [reply]
Mega wow.

**Reply by defib75** [reply]
Ultra wow

**Comment by regelkraut** [comment]
Ein weiterer Kommentar
